src/cljs/harja/ui/valinnat.cljs

```diff
@@ -19,20 +19,26 @@
    [livi-pudotusvalikko {:valinta    @valittu-sopimusnumero-atom
                          :format-fn  second
                          :valitse-fn valitse-fn
-                         :class      "suunnittelu-alasveto"
                          :li-luokka-fn #(when (= (first %) (:paasopimus ur))
                                           "bold")}
     (:sopimukset ur)]])
 
+(defn urakkatyyppi
+  [valittu-urakkatyyppi-atom urakkatyypit valitse-fn]
+  [:div.label-ja-alasveto.urakkatyyppi
+   [:span.alasvedon-otsikko "Urakkatyyppi"]
+   [livi-pudotusvalikko {:valinta    @valittu-urakkatyyppi-atom
+                         :format-fn  :nimi
+                         :valitse-fn valitse-fn}
+    urakkatyypit]])
+
 (defn urakan-hoitokausi
   [ur hoitokaudet valittu-hoitokausi-atom valitse-fn]
   [:div.label-ja-alasveto.hoitokausi
    [:span.alasvedon-otsikko (if (= :hoito (:tyyppi ur)) "Hoitokausi" "Sopimuskausi")]
    [livi-pudotusvalikko {:valinta    @valittu-hoitokausi-atom
                          :format-fn  #(if % (fmt/pvm-vali-opt %) "Valitse")
-                         :valitse-fn valitse-fn
-                         :class      "suunnittelu-alasveto"
```

Comment:
Tämä luokka on mielestäni turha, koska a) Sille ei löydy LESS-koodista toteutusta ja b) Näitä valintoja käytetään myös muualla kuin suunnittelupuolella, joten nimikin on varmaan väärin.